source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -0,0 +1,160 @@
+</pre>
+
+##### Request headers
+
+Header         | Value
+-------------- | -----
+`Content-Type` | `application/json: charset=utf-8`
+`User-Agent`   | `Skroutz OrderNotifier v1`
+
+##### Request payload
+
+Name         | Type   | Description
+:------------: | :------: | :----------------:
+`event_type` | String | Order event type (value: `"new_order"`)
+`order`      | Object | [Order details](#order-object)
+
+#### Order object
+
+Name         | Type   | Description
+:------------: | :-------: | :---------------:
+`code`       | String | Order code
+`comments`   | String | Order comments
+`line_items` | Array | [Order line items (products)](#order-line-items-array)
+`created_at` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order creation date
+`expires_at` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order expiration date
+`dispatch_until` | Date String in format<br> `YYYY-MM-DDTHH:MM:SS+HH:M` | Order maximum dispatch date
+
+#### Order line items array
+
+Name                              | Type      | Description
+:-------------------------------: | :-------: | :----------------------------------------------:
+`line_items[_].shop_uid`        | String  | Item id in shop
+`line_items[_].product_name`    | String  | Item product name
+`line_items[_].quantity`        | Integer | Item quantity
+`line_items[_].size`            | Object  | [Item size values (optional)](#line-item-size)
+`line_items[_].unit_price`      | Double  | Price per item in euros
+`line_items[_].total_price`     | Double  | Total item price in euros: `unit_price` * `quantity`
+
+#### Line item size
+
+Name                            | Type    | Description
+:-------------------------------: | :-------: | :------------------------------------------:
+`line_items[_].size.unit`       | String  | Item size unit
+`line_items[_].size.value`      | String  | Item size value
+`line_items[_].size.shop_value` | String  | Item original size as provided by the shop
+
+##### Payload Examples
+
+#### Example 1
+
+~~~ json
+{
+  "event_type": "new_order",
+  "order": {
+    "code": "191029-5130474",
+    "comments": "Παράδοση στο γραφείο",
+    "line_items": [
+        {
+          "shop_uid": "10",
+          "product_name": "Paul Mitchell Ultimate Color Repair 200ml",
+          "quantity": 1,
+          "unit_price": 25,
+          "total_price": 25
+        }
+    ],
+    "created_at": "2019-10-29T10:39:23+02:00",
+    "expires_at": "2019-10-29T16:39:23+02:00",
+    "dispatch_until": "2019-10-30T15:00:00+02:00"
+  }
+}
+~~~
+
+#### Example 2
+
+~~~ json
+{
+  "event_type": "new_order",
+  "order": {
+      "code": "191025-0111363",
+      "comments": "",
+      "line_items": [
+        {
+          "shop_uid": "405753",
+          "product_name": "adidas Perormance Badge of Sport Swimsuit PS/GS ( DQ3375 )",
+          "quantity": 2,
+          "size": {
+              "unit": "Ηλικία",
+              "value": "14 χρονών",
+              "shop_value": "15 χρονών"
```

Comment:
[minor] Maybe we could add a more prominent example of what this is about:

```
"unit": "Ηλικία",
"value": "14 χρονών",
"shop_value": "12-14"
```

Or:
```
"unit": "Νούμερο",
"value": "42",
"shop_value": "UK 8"
```